Serializer/SymfonySerializerAdapter.php

```diff
@@ -59,7 +59,9 @@ private function convertContext(Context $context)
             $newContext[$key] = $value;
         }
 
-        $newContext['groups'] = $context->getGroups();
+        if (count($context->getGroups())) {
```

Comment:
Is that what dunglas proposed in the bug comment (not sure if I understood well)?